pebbles/static/partials/account.html

```diff
@@ -34,7 +36,9 @@
                 <td>
                     <span ng-class="{user:'label label-default', manager:'label label-primary', owner:'label label-success'}[group.role]">{{group.role}}</span>
                 </td>
-                <td>{{group.description}}</td>
+		<td ng-show="group.role=='manager'"> {{group.join_code}} </td>
+		<td ng-show="group.role=='user'"> </td>
```

Comment:
Would be better that instead of having this, we can totally remove the join_code column for anyone who is just a normal user. The privileges are given to a user once he is appointed a group manager (now group admin). We are already returning that info in AuthService , I suppose, so you can make use of that.